hypervisor/arch/x86/guest/vcpu.c

```diff
@@ -677,26 +691,32 @@ void schedule_vcpu(struct acrn_vcpu *vcpu)
 	release_schedule_lock(vcpu->pcpu_id);
 }
 
+/*
+* pre &vcpu->stack[CONFIG_STACK_SIZE] shall align to 16
```

Comment:
Is this a valid @pre? Please check with Xiangyang?

Reply:
Only need to update format about pre-condition, this pre-condition is guaranteed by compiler and related attribute (__aligned).